play/PlayerRules/Basic Rules.txt

```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
-trigger, the gun fires and chambers a new round automatically one time. It 
+trigger, the gun fires once and chambers a new round automatically. It 
 takes 1 Action to fire a semi-auto weapon in this way. Some guns, 
 such as Shotguns and Bolt-Action sniper rifles require 2 Actions to fire 
 the gun, where the first Action is a trigger pull, and the second Action
 is the rechambering of the weapon.
-To hit, Players must roll a percentile that is over their weapon's % miss 
-stat. When an enemy is in cover, their cover score is added to your weapon's 
-miss chance. If your normal miss chance is 25, and the enemy is in full cover, 
-your miss chance is increased by 50, for a final DC of 75.
-Critical Hits:
-If your Luck is 5, on a roll of 90-1 through 90-5, the hit is considered a 
-Critical hit, and deals an extra 10 damage. on a roll of 90-6 through 00-0, 
-the hit is considered an Extra-Critical hit, and deals an extra 20 damage. 
+To hit a target, Players must roll a d10 that is over their weapon's miss stat.
+When an enemy is in cover, their cover score is added to your weapon's miss chance. 
+Lets say your normal ranged miss chance is 2, and the enemy is in full cover, your
+miss chance is increased by 4, for a final DC of 6. So a roll of 7-10 would be 
+successful.
 
-Your Luck modifier ((Luck-5)*4) is subtracted from the lower bound of your critical
-hit range and half of your luck modifier is subtracted from the lower bound of your
-extra-critical hit range. All weapons can critical hit except for Fully-Automatic
-weapons (weapons that shoot more than 1 bullet per half-action by default).
+Critical Hits:
+Critical hits are accuracy rolls that land inside your critical hit range (aka. crit chance). 
+If you roll a critical hit, your weapon does +20 damage for that attack (and/or other effects,
+depending on your abilities). By default, everyone starts with a critical hit range of "1", meaning, an
+attack roll of a 10 would be considered a critical hit. But your critical range may 
+not be "1" because your Luck modifies your critical hit range: ((Luck-5)*0.4), round up, is
+added to your critical hit range. Which means, if you have a Luck stat of 6 (and thus,
+by the equation, +1 to your critical hit range), an attack roll of 9-10 would be considered
+a critical hit. 
+All weapons and melee can critical hit except for Fully-Automatic weapons (weapons that 
+shoot more than 1 bullet per Action by default).
 
 Gun Jams:
-On a roll of 00-01, the weapon Jams, necessitating a weapon skill check, DC 60.
-When a player is defending against a hit, they roll Percentiles under their 
-armor in order to apply a Damage Reduction, or to apply damage to their 
-Capacitive Sheilds if the player has them.
+If at any time that you are firing a gun and you roll a weapon accuracy of 1, roll a second d10.
+If that second d10 roll is a 1 or 2, the weapon Jams, necessitating a weapon 
+skill check, DC 60. 

```

Comment:
I'm not sure what you mean... The intent of this was to exactly replicate the way jams are done in the percentile version. By, effectively, adding back in the one's place die roll.